deploy/helm/azure-industrial-iot/README.md

```diff
@@ -106,16 +106,13 @@ The following details of the Azure IoT Hub would be required:
 
     $ az iot hub consumer-group create --hub-name MyIotHub --name telemetry
 
-    $ az iot hub consumer-group create --hub-name MyIotHub --name tunnel
```

Comment:
Please change `Four` to `Three` or `Several` on line `101` as we have one less consumer group.